src/file.cpp

```diff
@@ -2050,6 +2050,7 @@ typedef struct _FILE_ALLOCATED_RANGE_BUFFER {
 			}
 		}
 
+#if _WIN32_WINNT >= 0x0600 // only if Windows Vista or newer
```

Comment:
I thought the whole point of manually loading this function out of the DLL was to not have to add this conditional compilation check. so that even if you build the program on XP, it will determine at run-time whether it has access to this function or not.

what inside this block will fail when building on/for XP?